packages/gatsby/src/utils/webpack.config.js

```diff
@@ -166,6 +166,7 @@ module.exports = async (
           // ids to reduce filesize.
           new webpack.NamedModulesPlugin(),
           new FriendlyErrorsWebpackPlugin({
+            clearConsole: false,
```

Comment:
I turned this off because the webpack output was clearing gatsby errors very quickly